Frends.CassandraDB.Execute/Frends.CassandraDB.ExecuteQuery.Tests/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+﻿version: '3'
```

Comment:
This file is not used in Workflows. Either remove this or update workflows to use docker compose.